documentation/book/assembly-kafka-exporter.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+[id='assembly-kafka-exporter-{context}']
+
+= Kafka Exporter
+
+The Kafka Exporter is provided with {ProductName} for deployment with a Kafka cluster to extract additional metrics from Kafka brokers related to offsets, consumer groups, consumer lag, and topics.
```

Comment:
One more comment ... can we provide a link to the project website (https://github.com/danielqsj/kafka_exporter)?As a courtesy to the author but also to explain the name etc. Maybe we can add some introduction. Something like this:

```suggestion
link:https://github.com/danielqsj/kafka_exporter[Kafka Exporter] is open source project which improves monitoring of Apache KAfka brokers and clients. 
Kafka Exporter is provided with {ProductName} for deployment with a Kafka cluster to extract additional metrics from Kafka brokers related to offsets, consumer groups, consumer lag, and topics.
```